source/nodejs/adaptivecards/src/serialization.ts

```diff
@@ -111,6 +116,22 @@ export class Versions {
     static readonly v1_5 = new Version(1, 5);
     static readonly v1_6 = new Version(1, 6);
     static readonly latest = Versions.v1_6;
+    // version over latest is considered "vNext feature"
+    // if latest or maxSupportedVersion is changed,
+    // don't forget to update styles .ac-schema-version-1-?::after to add/remove Preview tag from adaptivecards-site\themes\adaptivecards\source\css\style.css
+    static readonly maxSupportedVersion = Versions.v1_6;
```

Comment:
nit: why can't we `do static readonly maxSupportedVersion = latest` since the latest and maxSupportedVersion is same atm?

Reply:
It's a merge bug here. Likely Carousel changed this value, latest should be v1_5 here.